qds_sdk/cmd_line.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+class CmdLine:
```

Comment:
Why does it contain `SchedulerCmdLine` in the run method below? Are all the methods of this file used?